src/notification.js

```diff
@@ -2,165 +2,168 @@ import React, { Component } from 'react';
 import defaultPropTypes from './defaultPropTypes';
 
 class Notification extends Component {
```

Comment:
It looks like everything in this class was already two spaces. I don't think you need this diff.